modules/tinymce/CHANGELOG.md

```diff
@@ -253,6 +253,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## 5.4.0 - 2020-06-30
 
 ### Added
+- Added a new `toolbar_sticky_offset` setting to allow the toolbar to be offset from the top of the page #TINY-7337
```

Comment:
Don't forget to merge develop and move this to the `Unreleased` section.

- [ ] Move to correct changelog location